src/patternfly/components/DataList/data-list.scss

```diff
@@ -46,18 +46,18 @@
   --pf-c-data-list__item-item--before--Height: initial; // remove after breaking change
 
   // Data list item row
-  --pf-c-data-list__item-row--PaddingRight: var(--pf-global--spacer--lg);
-  --pf-c-data-list__item-row--md--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-data-list__item-row--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-data-list__item-row--md--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-data-list__item-row--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-data-list__item-row--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-data-list__item-row--md--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-data-list__item-row--md--PaddingLeft: var(--pf-global--spacer--lg);
 
-  @media screen and (max-width: $pf-global--breakpoint--md) {
+  @media screen and (min-width: $pf-global--breakpoint--md) {
     --pf-c-data-list__item-row--PaddingRight: var(--pf-c-data-list__item-row--md--PaddingRight);
     --pf-c-data-list__item-row--PaddingLeft: var(--pf-c-data-list__item-row--md--PaddingLeft);
   }
 
   // Data list item content
-  --pf-c-data-list__item-content--PaddingBottom: var(--pf-global--spacer--lg);
```

Comment:
You still have a reference to this var here that needs to change to `--pf-c-data-list__item-content--md--PaddingBottom`

https://github.com/patternfly/patternfly/blob/2c4b42c08dfec4be8a623678bf54297c9b39b19b/src/patternfly/components/DataList/data-list.scss#L168

Reply:
good catch thank you!